draft-quic-atsss-reqs.mkd

```diff
@@ -118,7 +118,11 @@ The two scenarios that we consider in this document are:
 - the failure of one access network that forces all data to switch to the other access network
 - the possibility of splitting the transmission of the data over the two access networks even if they have different characteristics (delay, bandwidth, congestion)
 
+## Discovery of the MPTCP Proxy
 
+During the attachment of an ATSSS-capable UE to the network, the UE retrieves the MPTCP proxy information: an IP address, a port number, and the type of the MPTCP proxy. In the current release, the mandatory MPTCP proxy type is the "Transport Converter" [I-D.ietf-tcpm-converters].
```

Comment:
Will future version of this document list a proxy discovery method as a requirement for ATSSS ? If not this detail can be left out.